app/services/support_interface/sites_export.rb

```diff
@@ -18,7 +18,10 @@ def sites
   private
 
     def relevant_sites
-      Site.includes(:provider).where(providers: { sync_courses: true }).order('providers.code ASC')
+      Site.includes(:course_option, :provider)
```

Comment:
`includes` will generate a `LEFT OUTER JOIN` in this case but I think you need an `INNER JOIN` so I think you might need a `joins(:course_options, :provider)` if I understand correctly (a big 'if').

Reply:
Yeah you're right - will make the change 👍

Reply:
```
[5] pry(main)> Site.joins(:course_options, :provider).where(course_options: { site_still_valid: true }).where(providers: { sync_courses: true }).order('providers.code ASC')
  Site Load (5.2ms)  SELECT "sites".* FROM "sites" INNER JOIN "course_options" ON "course_options"."site_id" = "sites"."id" INNER JOIN "providers" ON "providers"."id" = "sites"."provider_id" WHERE "course_options"."site_still_valid" = $1 AND "providers"."sync_courses" = $2 ORDER BY providers.code ASC  [["site_still_valid", true], ["sync_courses", true]]
  ```